### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Bludidi**
***Hello Reviewer!***
### In this milestone, I have implemented the following:
- [ ] Design from sneak peek wireframes.
- [ ] Used methods that I have created in the BlogApp processing data in models

### CODE REVIEW

File: db/migrate/20230223152516_add_default_value_to_counters.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddDefaultValueToCounters < ActiveRecord::Migration[7.0]
+  def change
+    change_column_default :users, :posts_counter, 0
+    change_column_default :posts, :comments_counter, 0
+    change_column_default :posts, :likes_counter, 0
+  end
+end
```
Comment (andersonlebon) [inline db/migrate/20230223152516_add_default_value_to_counters.rb]: image

- [ ] Kindly before running a migration that will modify a database table kindly make sure the table was created before. you can run the following CL to generate the migration that will create the above table you want to modifier and ensure the migration that will create the table comes first.

```zsh
rails g migration createUsers
rails g migration createPosts
```

File: db/migrate/20230224055950_remove_columns_from_posts.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class RemoveColumnsFromPosts < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :posts, :AuthorId, :integer
```
Comment (andersonlebon) [inline db/migrate/20230224055950_remove_columns_from_posts.rb]: image

- [ ] Kindly note that the `AuthorId` does not exist in any of your database records. kindly fix this.

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,17 @@
...
+fourth_user = User.create(name: 'Jb', photo: 'https://images.pexels.com/photos/39317/chihuahua-dog-puppy-cute-39317.jpeg', bio: 'Doctor from USA.')
+fifth_user = User.create(name: 'Wang', photo: 'https://images.pexels.com/photos/847393/pexels-photo-847393.jpeg', bio: 'Doctor from USA.')
+sixth_user = User.create(name: 'Bob', photo: 'https://images.pexels.com/photos/617278/pexels-photo-617278.jpeg', bio: 'Doctor from USA.')
+seventh_user = User.create(name: 'Xi', photo: 'https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg', bio: 'Doctor from USA.')
+
+first_post = Post.create(author: second_user, title: 'Hello', text: 'This is my first post')
+second_post = Post.create(author: first_user, title: 'Ola', text: 'I am the first user')
+
+Comment.create(post: first_post, author: first_user, text: 'Hi mary!' )
+Like.create(post: first_post, author: first_user )
```
Comment (andersonlebon) [inline db/seeds.rb:21]: - [ ] I am sorry but you need to review your database migrations, your database does not reflect the content of your project. You need to set up very well your migrations. if you need help on this kindly contact me on slack @andersonlebon

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Bludidi 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

almost-there

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/seeds.rb (hunk 3 above)
Comment (Bludidi) [reply db/seeds.rb:21]: @andersonlebon I am trying to reach out to you on slack but the username you gave seem to be incorrect

Comment (andersonlebon) [reply db/seeds.rb:21]: @Bludidi  Sorry for that, I miss spelled it, the right one is @Anderson Buyana

**Review by uwadonat — APPROVED** [review]
## Hi @Bludidi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Well organized and clen project ✔️
- Professional readme file ✔️
thanks-awesome

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**